frontend/app/pages/leaderboard.vue

```diff
@@ -91,6 +91,43 @@ const chartOptions = ref<ChartOptions<'line'>>({
 const { data: teams } = await useApi('/leaderboard/teams', {
   key: 'leaderboard-teams',
 })
+
+const teamsTableData = ref(teams.value ?? [])
+
+async function updateLeaderboard() {
+  const { $api } = useNuxtApp()
+
+  const chartDataResponse = await $api('/leaderboard/chart')
+  const teamsResponse = await $api('/leaderboard/teams')
+
+  const adjustedTimestamps = chartDataResponse.event_timestamps.map((timeStamp: string) =>
```

Comment:
A lot of duplicated logic, for example this function. There is no need to send all request again, just use `refresh` method provided by `useFetch`, thanks to everything being reactive it should work proerly